herciuana_Task4/new.hpp

```diff
@@ -0,0 +1,19 @@
+#include <chrono>
+#include <string>
+#include <iostream>
+class Timer
+{
+private:
+    std::chrono::time_point<std::chrono::steady_clock> StartTime;
+
+public:
+    Timer() : StartTime{std::chrono::steady_clock::now()}
+    {
+    }
+    void Start(std::string str = "")
```

Comment:
The name of the method is misleading, please use another name(printDiff, showDiff, calculateTime, etc.)